site/en/docs/extensions/upgrading/to-service-workers/index.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Upgrade to a service worker
+subhead: 'Replacing background or event pages with a service worker'
+description: 'A service worker replaces background or event pages to ensure that background code stays off the main thread. This enables extensions to run only when needed, saving resources.'
```

Comment:
> description: 'A service worker replaces background or event pages to ensure that background code stays off the main thread. This enables extensions to run only when needed, saving resources.'

It might be a good idea to condense this a bit. 

<img width="312" alt="Screenshot 2023-03-06 at 12 14 12" src="https://user-images.githubusercontent.com/37001393/223195718-7fb5bfff-d66f-4ca2-9811-9d166ae69369.png">

Reply:
I shrunk this.